pkg/cli/cli.go

```diff
@@ -176,8 +178,19 @@ func NewOcCommand(o kubecmd.KubectlOptions) *cobra.Command {
 		Long:  cliLong,
 		Run:   runHelp,
 		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
-			rest.SetDefaultWarningHandler(warningHandler)
-
+			switch cmd.Name() {
```

Comment:
I'd not prefer modifying this core cli entrance point just to update the warning text.